/**:
    ros__parameters:
        common:
            lid_topic:  "/livox/lidar" 
            # imu_topic:  "/imu/data"  #from imu_complementary_filter
            imu_topic:  "/livox/imu" 
            con_frame: false # true: if you need to combine several LiDAR frames into one
            con_frame_num: 1 # the number of frames combined
            cut_frame: false # true: if you need to cut one LiDAR frame into several subframes
            cut_frame_time_interval: 0.1 # should be integral fraction of 1 / LiDAR frequency
            time_lag_imu_to_lidar: 0.0 # Time offset between LiDAR and IMU calibrated by other algorithms, e.g., LI-Init (find in Readme)
                                    # the timesample of IMU is transferred from the current timeline to LiDAR's timeline by subtracting this value
            

        preprocess:
            lidar_type: 1  
            scan_line: 4
            scan_rate: 10
            timestamp_unit: 3           
            blind: 0.1 

        mapping:
            imu_en: true
            start_in_aggressive_motion: false 
            extrinsic_est_en: false 
            imu_time_inte: 0.005 
            satu_acc: 3.0 
            satu_gyro: 35.
            acc_norm: 1.0
            lidar_meas_cov: 0.001
            acc_cov_output: 500.
            gyr_cov_output: 1000. 
            b_acc_cov: 0.0001 
            b_gyr_cov: 0.0001 
            imu_meas_acc_cov: 0.1
            imu_meas_omg_cov: 0.1
            gyr_cov_input: 0.01 
            acc_cov_input: 0.1 
            plane_thr: 0.1 
            match_s: 81.0
            fov_degree: 360. 
            det_range: 450.0
            gravity_align: false 
            gravity: [0.0, 0.0, -0.9810] 
            gravity_init: [0.0, 0.0, -0.9810] 
            extrinsic_T: [ -0.011, -0.02329, 0.04412 ]
            extrinsic_R: [  1., 0., 0.,
                            0., 1., 0.,
                            0., 0., 1.]    
        odometry: 
            publish_odometry_without_downsample: false
        publish:
            path_en: true                 
            scan_publish_en: true         
            scan_bodyframe_pub_en: true  

        pcd_save:
            use_pcd_map_: true
            prior_PCD_map_path: "/home/auto/test_rmuc_17.pcd" #gongxun
            # prior_PCD_map_path: "/home/auto/github_code/auto_sentry_ws/file/test_rmuc_17.pcd" #RMUC
            #/home/auto/github_code/auto_sentry_ws/file/gcl_3.pcd /home/auto/code/PCD/gongxun_13.pcd
            #/home/auto/code/PCD/test_no_wall_2.pcd  /home/auto/code/PCD/scans_3v3.pcd /home/auto/code/PCD/scans_3v3_rot_rem.pcd test_no_wall_2 test_no_wall_2_only_edge
            #test_no_wall_3 /home/auto/code/PCD/gongxun_1.pcd /home/auto/github_code/auto_sentry_mid360_ws_simple/file/5v5_5.pcd
            pcd_save_en: true
            interval: -1                  
        # init_pos:
        #     # init_x: 6.0
        #     # init_y: -0.3
        #     # init_z: 0.0
        #     # init_x: 0.0
        #     # init_y: 0.0
        #     # init_z: 0.0
        #     # init_x: 0.97
        #     # init_y: 2.34
        #     # init_z: 0.0
        #     init_x: 0.0
        #     init_y: -3.0
        #     init_z: 0.0
        # gcl:
        #     task: "B"

        # init_pos:
        #     # init_x: 6.0
        #     # init_y: -0.3
        #     # init_z: 0.0
        #     # init_x: 0.0
        #     # init_y: 0.0
        #     # init_z: 0.0
        #     init_x: 13.417832374572754
        #     init_y: -2.940614938735962
        #     init_z: 0.0
        # gcl:
        #     task: "A"
        
        #RMUC    
        init_pos:
            # init_x: 6.0
            # init_y: -0.3
            # init_z: 0.0
            # init_x: 0.0
            # init_y: 0.0
            # init_z: 0.0
            init_x: 0.97
            init_y: 2.34
            init_z: 0.0
            # init_x: 0.0
            # init_y: 0.0
            # init_z: 0.0
        gcl:
            task: "B"